File: python/washing_machine.py

def calculate_washing_time(weight):
    
        weight = int(weight)

        # Calculate washing time based on weight
        if weight == 0:
            return "0 minutes"
        elif 0 < weight <= 2000:
            return "25 minutes"
        elif 2001 <= weight <= 4000:
            return "35 minutes"
        elif 4001 <= weight <= 7000:
            return "45 minutes"
        else:
            return "Invalid input"
    


weight = input("Enter the weight : ")
print(calculate_washing_time(weight))

File: C++/Washing_machine.cpp
#include <iostream>
#include <string>
using namespace std;

string calculate_washing_time(int weight) {
    if (weight == 0) {
        return "0 minutes";
    } else if (weight > 0 && weight <= 2000) {
        return to_string(25) + " minutes";
    } else if (weight >= 2001 && weight <= 4000) {
        return to_string(35) + " minutes";
    } else if (weight >= 4001 && weight <= 7000) {
        return to_string(45) + " minutes";
    } else {
        return "Invalid weight";
    }
}

int main() {
    int weight;
    cout << "Enter weight of clothes: ";
    cin >> weight;
    cout << calculate_washing_time(weight) << std::endl;
    return 0;
}